src/model.ts

```diff
@@ -266,7 +266,10 @@ export class Model extends owl.core.EventBus implements CommandDispatcher {
 
   private onRemoteRevisionReceived({ commands }: { commands: CoreCommand[] }) {
     for (let command of commands) {
+      const previousStatus = this.status;
+      this.status = Status.RunningCore;
```

Comment:
this and the `isReplayingInitialRevisions` check in session actually serves the same purpose.
With this, the error in session will never occur because `A UI plugin cannot dispatch while handling a core command` will be thrown first.
Should I still keep the check in session then? Just in case

Reply:
Hmm it breaks some tests

Reply:
Edition plugin dispatches `STOP_EDITION` when handing remove col/row removed...